### DISCUSSION TITLE
Web OSM alternative to Google/Apple maps

### DISCUSSION THREAD

**Discussion by laem** [organicmaps/organicmaps#9041] [Ideas]
Hi, I'm the main developer of https://cartes.app, a Web-only alternative to Google and Apple Maps. 

![image](https://github.com/user-attachments/assets/20adfd93-9f6d-4924-9739-5e959bac9de6)

It's a french-only project (France perimeter / French language), but I thought it would be a good thing to share it here, since the code is 100 % open source and written mostly in English. The map works out of France but with more generic features and base maps.

The rationale is simple : I believe we need a better Web OSM frontend. What has been done by Organic Maps is great, I use it often. But I also need a universal desktop/mobile Web version, that is more dynamic, plug and play (no offline download) and "live" (e.g. the weather widget). 

> Why only French ? Because i18ning an app is an often underestimated challenge. We non-english natives usually have to use apps with awkward translations and internationalisation, something that is intrinsic to low-budget teams, which Cartes totally is. Building an alternative to Gmaps cannot rely solely on OSM data and international conventions. For instance, we need premium transit coverage, which is for now country-specific. The task of integrating at once France's and Mexico's transit networks is too hard, it would result in two half-baked integrations. So we're starting to cover France region by region / town by town. Last but not least : the active community of testers and contributors has French as its native language. 

The first step for me is to get public traction in France first (remember : mapy.cz has 20 million users in Czech Republic alone). If it's a success, we could talk about extending the perimeter ! 

Cartes won't have any offline nor navigation mode in 2024. Hence the idea of downloading a GPX for mobile use and letting the user chose an open source mobile app to open the GPX, and Organic Maps would be the best fit I guess. We could also recommend Organic Maps on a page explaining why no mobile app.

**Comment by biodranik** [comment]
Hi, and thanks for spreading the word about your project in our discussions :) We would be happy to see recommendations to install Organic Maps on your "why no mobile app" page.

There is a plan to add a web browser experience in addition to the offline maps experience later, for easier/smoother UX. Maybe your project can be a base for such integration in the future, who knows? Of course, we're targeting the worldwide user base, and it should work compared or better than google maps )

**Reply by laem** [reply]
Yes, the page does not exist yet. My plan would be to integrate an popup on the right places, the right times to inform the user about those interesting open source mobile apps.
